Shopify/cli
# Cache user info

### WHY are these changes introduced?
We want to cache data that rarely changes and avoid the extra API calls

Fixes https://github.com/Shopify/develop-app-inner-loop/issues/2026



### WHAT is this pull request doing?
- Cache the `hasPartnerAccount` value. For a given userId, cache it forever if true.
- Cache user account information (email, etc..) for 72h. We were fetching this on every command, but rarely changes.
  - Created a new `app-conf-store.ts` to handle this 


### How to test your changes?
Running `app info --verbose` should show a call to fetch an organization and another to fetch the user info.
If you run it again, you should see that those queries are no longer executed (we are reading from cache)



### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
